### PR TITLE
Feature/bc 999 update status pages

### PR DESCRIPTION
**Pull request by MartinSchuhmacher**
# Description

As a user, I want to be directed to the new status page when I click the link "Status" in the footer.

## Links to Tickets or other pull requests

Ticket: https://ticketsystem.hpi-schul-cloud.org/browse/BC-999
Nuxt-PR: https://github.com/hpi-schul-cloud/nuxt-client/pull/2013
Server-PR: https://github.com/hpi-schul-cloud/schulcloud-server/pull/3259
Dev-PR: https://github.com/hpi-schul-cloud/dof_app_deploy/pull/82

## Changes

- environment variable set to default (dBildungscloud) and individually set for each instance
- small wording change in contact area with mentioned status page

## Deployment

ALERT_STATUS_URL: https://status.INSTANCE.de

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: ansible/roles/schulcloud-client-core/templates/configmap.yml.j2 [hunk 1]
```diff
@@ -106,8 +106,8 @@ data:
   GLOBAL_ANNOUNCEMENT_ADMIN_TEACHER: "{{ GLOBAL_ANNOUNCEMENT_ADMIN_TEACHER|default("", true) }}"
 {% endif %}
   FEATURE_ALERTS_ENABLED: "{{ FEATURE_ALERTS_ENABLED|default("false") }}"
-  ALERT_STATUS_URL: "{{ ALERT_STATUS_URL|default("https://status.hpi-schul-cloud.de", true) }}"
-  ALERT_STATUS_API_URL: "{{ ALERT_STATUS_URL|default("https://status.hpi-schul-cloud.de", true) }}/api/v1"
+  ALERT_STATUS_URL: "{{ ALERT_STATUS_URL|default("https://status.dbildungscloud.de", true) }}"
+  ALERT_STATUS_API_URL: "{{ ALERT_STATUS_URL|default("https://status.dbildungscloud.de", true) }}/api/v1"
```
Comment (tofixx) [inline ansible/roles/schulcloud-client-core/templates/configmap.yml.j2:110]: the api url might not be in use from the client...

**Review by tofixx — DISMISSED** [review]

File: ansible/roles/schulcloud-client-core/templates/configmap.yml.j2 (hunk 1 above)
Comment (MartinSchuhmacher) [reply ansible/roles/schulcloud-client-core/templates/configmap.yml.j2:110]: So it is not necessary or why was it there in the first place?

Comment (MartinSchuhmacher) [reply ansible/roles/schulcloud-client-core/templates/configmap.yml.j2:110]: If not necessary and nobody contradicts I would remove the API URL from the configmap in legacy client

Comment (Metauriel) [reply ansible/roles/schulcloud-client-core/templates/configmap.yml.j2]: if you cant find a mention of it in the client code, get rid of it.

**Review by Metauriel — APPROVED** [review]

**Pull request merged**